### PR TITLE
Add ActivateLocation screen to onboarding for required Android devices

### PR DESCRIPTION
**Pull request by devinjameson**
Why: some Android devices need Location to be on in Settings in order for Exposure Notifications to work.

This commit:
- Adds an `ActivateLocation` screen to the `ActivationStack` if the device needs location on for exposure notifications to work
- Navigates to the `ActivateLocation` screen from the `ActivateProximityTracing` screen only if location is off and needed for exposure notifications to work

Co-Authored-By: Amanda Beiner <aebeiner@gmail.com>

Screen Shot 2020-09-01 at 1 58 56 PM

### CODE REVIEW

File: src/navigation/ActivationStack.tsx [hunk 1]
```diff
@@ -104,7 +119,7 @@ const ActivationStack: FunctionComponent = () => {
 
   return (
     <Stack.Navigator screenOptions={screenOptions}>
-      {activationSteps.map((step, idx) => {
+      {activationSteps?.map((step, idx) => {
```
Comment (johnschoeman) [inline src/navigation/ActivationStack.tsx:123]: Whats the need for the optional operator here? wouldnt activationSteps always be defined as an array?

File: src/locales/ar.json [hunk 2]
```diff
@@ -22,6 +22,9 @@
     "launch_get_started": "ابدأ",
     "launch_screen1_header": "طريق العودة إلى الوضع الطبيعي يبدأ من هنا"
   },
+  "onboarding": {
```
Comment (johnschoeman) [inline src/locales/ar.json:25]: why are all these translations included in this pr? should they be in a different commit/pr?

File: src/Activation/ActivateProximityTracing.tsx [hunk 3]
```diff
@@ -59,6 +62,10 @@ const ActivateProximityTracing: FunctionComponent = () => {
     )
   }
 
+  const handleOnPressActivateProximityTracing = () => {
+    showProximityTracingAlert()
```
Comment (johnschoeman) [inline src/Activation/ActivateProximityTracing.tsx:66]: 💯

File: src/navigation/ActivationStack.tsx (hunk 1 above)
Comment (devinjameson) [reply src/navigation/ActivationStack.tsx:123]: For some reason Typescript thinks it could be undefined. Is there a workaround for that?

Comment (johnschoeman) [reply src/navigation/ActivationStack.tsx:123]: I would look into the type signature of Platform.select and see if theres a thread to follow there.

Comment (devinjameson) [reply src/navigation/ActivationStack.tsx:123]: Good call, it needed a default.

File: src/locales/ar.json (hunk 2 above)
Comment (devinjameson) [reply src/locales/ar.json]: Removed thank!
